### PR TITLE
Changes from audit

### PR DESCRIPTION
**Pull request by sunbreak1211**

### CODE REVIEW

File: test/funnels/callees/SwapperCalleeUniV3.t.sol [hunk 1]
```diff
@@ -48,11 +49,18 @@ contract SwapperCalleeUniV3Test is DssTest {
...
+        checkStableSwap(USDT, DAI, USDT_DAI_PATH);
+        checkStableSwap(USDT, DAI, USDT_DAI_PATH); // swapping a 2nd time to verify that USDT allowance has been cleared after the 1st swap
+    }
+
     function testSwapShortPath() public {
         bytes memory DAI_USDC_PATH = abi.encodePacked(DAI, uint24(100), USDC);
         checkStableSwap(DAI, USDC, DAI_USDC_PATH);
     }
 
+
```
Comment (telome) [inline test/funnels/callees/SwapperCalleeUniV3.t.sol]:
```suggestion
```

**Review by telome — APPROVED** [review]
LGTM

**Comment by sunbreak1211** [comment]
PR looks good to me, @oldchili please leave the green check and I'll squash and merge or feel free to do it yourself.

**Review by oldchili — APPROVED** [review]
LGTM

**Pull request merged**